### ISSUE TITLE
App is restart when i select the image from the camera.

### ISSUE THREAD

**Issue by gulsher7** [ivpusic/react-native-image-crop-picker#1519]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker 0.9.3
- react-native 0.63.4

### Platform
Tell us to which platform this issue is related
- Android(Redmin Note 7 Pro)

App is work fine in case of gallery selection image. But when i click the image from the camera then the app is restart.

**Comment by prtkkmrsngh** [comment]
@gulsher7 hey, we are also facing same issue with OPPO and VEVO devices. Did you find some workaround on this?

**Comment by akanksha-agarwal93** [comment]
Hello, we are facing the same issue with few Samsung devices. Did you guys find any solution on this ?

**Comment by hagar0459** [comment]
i also have the same issue inn huawei tablet ,is there any alternative option to get images form the gallery and not have  this issue,
my app is in production and need quick help on that ,please

**Comment by lewisbird** [comment]
Still also having the issue with some android devices though cannot spot a common pattern as to why. Not getting any react native errors either.

**Comment by mrsasuu** [comment]
Same here

**Comment by mrsasuu** [comment]
Can be related to #1509 and #1526?

**Comment by akanksha-agarwal93** [comment]
I got the issue resolved - 
for me, flipper was the problem. If you are using React-Native version > 0.60 try -

@override
public void onCreate() {
super.onCreate();
SoLoader.init(this, /* native exopackage */ false);
// initializeFlipper(this, getReactNativeHost().getReactInstanceManager()); -------> commenting this line helped
}

in MainApplication.java

**Comment by radadiyarushita** [comment]
Hello Team,
Same issue I've facing right now.
The app is restart when I select the image from the camera

**Comment by felix4321** [comment]
I have the same problem with an HUAWEI P30 lite. Are there any updates?

"react-native-image-crop-picker": "0.36.0",
"react-native": "0.64.0",

**Comment by syntacticsolutions** [comment]
Looks like the issue might be because of this https://stackoverflow.com/a/28246350/5231528 ???

**Comment by syntacticsolutions** [comment]
In my case the issue was that another module(react-native-stripe) was eating all of my intents and then starting new intents. Apparently React-Native bundles all of the native modules into a single native module so you have to check who the activity referrer is when calling your intent. I'm planning to use `patch-package` today to check the referrer when `onActivityResult` is called in react-native-stripe and not execute the logic unless it's coming from itself. Also going to file a bug with react-native-stripe to let them know this is happening.

**Comment by gulsher7** [comment]
i don't know what is going on why this library developer not fixed this issue.

**Comment by gulsher7** [comment]
> @gulsher7 hey, we are also facing same issue with OPPO and VEVO devices. Did you find some workaround on this?

yes i found some workaround click this link https://github.com/ivpusic/react-native-image-crop-picker/issues/1519#issuecomment-839804273

**Comment by gulsher7** [comment]
> In my case the issue was that another module(react-native-stripe) was eating all of my intents and then starting new intents. Apparently React-Native bundles all of the native modules into a single native module so you have to check who the activity referrer is when calling your intent. I'm planning to use `patch-package` today to check the referrer when `onActivityResult` is called in react-native-stripe and not execute the logic unless it's coming from itself. Also going to file a bug with react-native-stripe to let them know this is happening.

no its not another module problem. its only react-native-image-crop-picker problem. when you set the value of cropping true and compressImageQuality then app not restart.

**Comment by gulsher7** [comment]
> Still also having the issue with some android devices though cannot spot a common pattern as to why. Not getting any react native errors either.

if found some workaround try this method https://github.com/ivpusic/react-native-image-crop-picker/issues/1519#issuecomment-839804273

**Comment by felix4321** [comment]
@gulsher7 In my case the issue was related to another module (react-native-splash-screen)

**Comment by gulsher7** [comment]
> @gulsher7 In my case the issue was related to another module (react-native-splash-screen)

so how did you resolved it ?

**Comment by tarouboy** [comment]
probably related
https://github.com/ivpusic/react-native-image-crop-picker/issues/1557#issuecomment-962878316

**Comment by saeedtkh** [comment]
any updates?

**Comment by DaviAugustoMS** [comment]
Has anyone found a solution to this problem, I have this for the Samsungs

**Comment by c-info** [comment]
Having the same issue on Readmi Note 7 pro with the below configuration on picker

{"avoidEmptySpaceAroundImage": true, "compressImageMaxHeight": 1200, "compressImageMaxWidth": 1200, "compressImageQuality": 0.7, "cropperActiveWidgetColor": "#F90606", "cropperCircleOverlay": true, "cropperStatusBarColor": "#FFFFFF", "cropperToolbarColor": "#FFFFFF", "cropperToolbarWidgetColor": "#F90606", "cropping": true, "freeStyleCropEnabled": true, "includeBase64": false, "mediaType": "photo", "showCropFrame": true, "showCropGuidelines": true}

using 

```
"react": "18.2.0",
"react-native": "0.74.3",
"react-native-image-crop-picker": "^0.41.1",
```
